app/models/trainee.rb

```diff
@@ -40,6 +40,7 @@
 #  itt_end_date                    :date
 #  itt_start_date                  :date
 #  last_name                       :text
+#  lead_partner_not_applicable     :boolean          default(FALSE), not null
```

Comment:
I'm working on renaming `lead_school_not_applicable` to `lead_partner_not_applicable` as part of [7354-implement-itt-reform-on-register-api](https://trello.com/c/gMzDEEyF/7354-implement-itt-reform-on-register-api)